.gitignore

```diff
@@ -8,13 +8,21 @@ tags
 .classpath
 .project
 local.properties
-
+.gradleops/AndroidSeed/idea
+.gradleops/AndroidSeed/idea/gradle.xml
+.gradleops/AndroidSeed/idea/dictionaries		  
+.gradleops/AndroidSeed/idea/dictionaries
+.gradleops/AndroidSeed/idea/libraries		  
+.gradleops/AndroidSeed/idea/libraries
+ +.idea/misc.xml
 # build directories
 bin
 gen
-obj
-armeabi
-armeabi-v7a
+
+#Removed for now
```

Comment:
Why have these been commented out ?

Reply:
Give me more details, please.

Reply:
Lines 20-23 have been commented out.
Was checking why and should they be uncommented ?

Reply:
Lines 20-22 have been commented out and your comment says
"Removed for now"
Should we put it back ?

Reply:
Alex, is this still okay ?